src/typings/browser.d.ts

```diff
@@ -106,6 +106,17 @@ interface HTMLVideoElement {
 }
 interface HTMLMediaElement {
   webkitGenerateKeyRequest? : (key: string, initData : ArrayBuffer) => void;
+  webkitDroppedFrameCount?: number;
+  webkitDecodedFrameCount?: number;
+  getVideoPlaybackQuality?: () => IVideoPlaybackQuality;
+}
+
+interface IVideoPlaybackQuality {
```

Comment:
This is a rx-player-specific interface, so no reason for it to be included in ``browser.d.ts``

Reply:
Not really a specific RxPlayer interface https://developer.mozilla.org/en-US/docs/Web/API/VideoPlaybackQuality